packages/allow-scripts/src/toggles.js

```diff
@@ -0,0 +1,3 @@
+module.exports = {
+  FT: Object.create(null)
```

Comment:
Can this get a more descriptive name?

Reply:
I stopped myself too early, yes.

My favorite format of feature toggle module is a function that accepts:
```js
featureToggle('feature-name', {
on: ()=>{},
off: ()=>{}
})
```
And demands all keys in use to be explicitly set before use.
I thought it'd be a bit of an overkill. Gotta decide where on the specrtum to stop ;)   
Counting your comment as "not _this_ early".

Reply:
My 5c: Something like this keeps the current simplicity and makes intentions more clear. Some jsdoc typings would make it sufficient? 

```
module.exports = {
  FeatureToggled: {
    bins: undefined, // or if you want to put default value here already
  },
};
```

Reply:
I put the default value in and made sure all toggles need to be defined upfront in the file. how do you like that?

Reply:
Much better, and nice usage of `Object.seal` :)